### ISSUE TITLE
BUS.fd_baudrate in dbc.py "_load_bus" function 

### ISSUE THREAD

**Issue by BoningLiao** [cantools/cantools#524]
We want to grab fb_baudrate through cantools in the dbc file, but the return value is None, trace the source code we found the _load_bus function doesn't check the fd_baudrate, and we add some code on our side, and it works.

**Comment by andlaus** [comment]
yes, the DBC parser currently does not read the FD baudrate (only the [normal signalling rate](https://github.com/cantools/cantools/blob/master/cantools/database/can/formats/dbc.py#L1681-L1684); as far as I know, the only file format which currently reads the FD baudrate is ARXML). if you have code for that, we would be grateful if you contributed it...
